meson.build

```diff
@@ -45,7 +45,9 @@ gio_dep = [dependency('gio-2.0', version: glib_req_version,
                   fallback: ['glib', 'libgio_dep'])]
 gobject_dep = dependency('gobject-2.0', version: glib_req_version,
     fallback: ['glib', 'libgobject_dep'])
-gstreamer_dep = dependency ('gstreamer-1.0', version: '>= 1.16.0')
+gstreamer_dep = dependency ('gstreamer-1.0', version: '>= 1.14.0')
```

Comment:
Actually, we are going to use gstreamer >= 1.16, but ubuntu 18.04 doesn't have it unless we do some backporting effort. In the meantime, this is okay to pass CI.